EasyIntern-Backend/Models/JsonModelAccountLogin.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace EasyIntern_Backend.Models;
+
+public class JsonModelAccountLogin
+{
+    [Required]
+    public string Email { get; set; }
```

Comment:
Add the [EmailAddress] Attribute to validate the email to be an email adress

Reply:
Again, not relevant to the login dto